plainly-plugin/src/ui/components/projects/LinkedProject.tsx

```diff
@@ -68,6 +71,13 @@ export function LinkedProject({
     [project.id, openInWeb],
   );
 
+  const openRenders = useCallback(
+    () => openProjectRenders(project.id),
+    [project.id, openProjectRenders],
+  );
+
+  const templatesLength = useMemo(() => project.templates.length, [project]);
```

Comment:
please remove `useMemo`... the purpose of this hook is to memoize heavy calculations, this is unnecessary overhead, `project.templates.length` is instant